pkg/transaction/transaction.go

```diff
@@ -48,6 +48,8 @@ type TxRequest struct {
 	Data        []byte          // transaction data
 	GasPrice    *big.Int        // gas price or nil if suggested gas price should be used
 	GasLimit    uint64          // gas limit or 0 if it should be estimated
+	GasTipBoost int             // adds a tip for the miner for prioritizing transaction
```

Comment:
so i dont think we need to store this
what we need is the gasTipCap from the transaction